scripts/compare_testsuite_log.py

```diff
@@ -316,7 +324,7 @@ def compare_testsuite_log(previous_log_path: str, current_log_path: str):
     return classified_gcc_failures
 
 
-def gccfailure_to_summary(failure: Dict[LibName, GccFailure], failure_name: str, previous_hash: str, current_hash: str):
+def gccfailure_to_summary(failure: Dict[LibName, GccFailure], failure_name: str, previous_hash: str, current_hash: str, current_hash_committed: bool):
```

Comment:
Is this for precommit if the patch is already merged? If so, how do we determine that?

Reply:
gcc-postcommit-ci will set that to yes, precommit will conservatively say it's not already committed